assignments/arrays.js

```diff
@@ -63,34 +63,59 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+for(car in inventory){
+  if(inventory[car]["id"] === 33){
+    console.log(`Car ${inventory[car]["id"]} is a ${inventory[car]['car_year']} ${inventory[car]['car_make']} ${inventory[car]['car_model']}`)
+  }
+}
+
 
 
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+let lastCar = inventory.length;
+for(car in inventory){
```

Comment:
Similar to your first solution this definitely gets it done but you could have done it in a simpler way.
```
let lastCar = inventory[inventory.length -1];
console.log(lastCar);
```
No need to loop and create extra run time.